src/main/java/org/fungover/mmotodo/team/TeamController.java

```diff
@@ -0,0 +1,77 @@
+package org.fungover.mmotodo.team;
+
+import jakarta.validation.Valid;
+import org.fungover.mmotodo.team.Team;
+import org.fungover.mmotodo.team.TeamDto;
+import org.fungover.mmotodo.team.TeamService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.graphql.data.method.annotation.Argument;
+import org.springframework.graphql.data.method.annotation.MutationMapping;
+import org.springframework.graphql.data.method.annotation.QueryMapping;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import java.util.List;
+
+@Controller
+public class TeamController {
+    private final TeamService teamService;
+
+    @Autowired
```

Comment:
The @Autowired is not needed in Spring Boot.

Reply:
Well,technically it is not wrong because it is most useful when we have more than one constructor. It can be omitted as well as included